lib/interfaces/bdd.js

```diff
@@ -1,7 +1,18 @@
 "use strict";
 
 var Test = require("./common").Test,
-    _ = require("../extended");
+    _ = require("../extended"),
+    descriptions = {},
+    duplicateDescribeBlockErrors = [];
+
+
+_.bus.on("checkDuplicates", function (duplicateTests) {
+    duplicateDescribeBlockErrors = duplicateDescribeBlockErrors.concat(duplicateTests);
```

Comment:
I think we could just add this as a static method on Test and invoke inside of `run` instead of emitting an event inside of run and just skip all tests and emit the `printDuplicateActions` event instead of throwing an error since the stack wont do much good here.